Tests/interop/net/test_loadorder.py

```diff
@@ -1,124 +0,0 @@
-#####################################################################################
```

Comment:
I probably asked this in the original PR, but why is this file getting removed? I'm not seeing a replacement.

Reply:
Here's the comment for when I did it in ipy2

https://github.com/IronLanguages/ironpython2/pull/2#discussion_r119403155